sklearn/metrics/cluster/tests/test_common.py

```diff
@@ -126,7 +123,7 @@ def test_normalized_output(metric_name):
 # that is when 0 and 1 exchanged.
 @pytest.mark.parametrize(
     "metric_name",
-    [name for name in dict(SUPERVISED_METRICS, **UNSUPERVISED_METRICS)]
```

Comment:
Someone has pointed out that ``dict(x, **y)`` is not recommended in part1. But I don't think we're going to modify parametrized tests too much, so LGTM.